src/lib/journey/callbacks/device-profile/device-profile.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import type { SelfSubmitFunction, StepMetadata } from '$journey/journey.interfaces';
+  import { FRDevice, type DeviceProfileCallback } from '@forgerock/javascript-sdk';
+  import type { Maybe } from '$lib/interfaces';
+  import type { ProfileConfigOptions } from '@forgerock/javascript-sdk/lib/fr-device/interfaces';
+  import Spinner from '$components/primitives/spinner/spinner.svelte';
+
+  export let spinner = false;
+  export let callback: Maybe<DeviceProfileCallback> = null;
```

Comment:
If this component is called, the `callback` type _is_ `DeviceProfileCallback`, so the `Maybe` is not needed.